src/templates/modulelist.jsx

```diff
@@ -44,7 +44,13 @@ const modulelist = (
         DATA.filter((module) => {
           const searchRgx = new RegExp(escapeRegExp(searchQuery), "gi");
           const tagRgx = new RegExp(escapeRegExp(filterTag), "gi");
-          return module.tags.match(tagRgx) && module.title.match(searchRgx);
+          let matchedTag = [];
+          module.tags.map((tag) => {
+            if (tag != null) {
+              matchedTag.push(tag.match(tagRgx));
+            }
+          });
+          return matchedTag.toString().match(tagRgx) && module.title.match(searchRgx);
```

Comment:
```suggestion
          const matchedTag = blog.tags.filter(tag => tag != null).map(t => t.match(tagRgx))
          return matchedTag.toString().match(tagRgx) && blog.title.match(searchRgx);
```
I think you can do something like this. your using a map wrong with this expression. map is suppose to map A --> B. javascript does have a foreach prototype if I remember correctly.

Reply:
done

Reply:
> done , foreach will not return modified array